app/notifications/callbacks.py

```diff
@@ -0,0 +1,60 @@
+from app.config import QueueNames
```

Comment:
Nothing new in this file, just reorganized the code because some logic is shared between SNS and SES Celery tasks